benches/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "benches"
 version = "0.0.0"
 publish = false
 edition = "2021"
+license = "MIT"
```

Comment:
Are these needed for `publish = false` crates? I thought they did not.

Reply:
It seems like you can turn on an option so they're not needed, but that is not the case by default. I think it's fine to just add them :shrug:

Reply:
Fine with me, I usually enable that option IIRC.